charts/sysdig/templates/daemonset-node-analyzer.yaml

```diff
@@ -381,7 +381,7 @@ spec:
                 name: {{ template "sysdig.fullname" . }}-runtime-scanner
                 key: analyzer.maxFileSizeAllowed
                 optional: true
-          - name: SYSDIG_SECURE_ENDPOINT
+          - name: SYSDIG_API_URL
             valueFrom:
               configMapKeyRef:
                 name: {{ template "sysdig.fullname" . }}-runtime-scanner
```

Comment:
let's change the configmap key below (now `secure_url`) as well, for consistency